library/network/src/modules/SuSEFirewall.rb

```diff
@@ -38,6 +38,9 @@ class SuSEFirewallClass < Module
 
     include Yast::Logger
 
+    YES = "yes"
+    NO  = "no"
```

Comment:
better way is to create mapping between boolean like
```
INIT_MAPPING = {
  true  => "yes",
  false => "no"
}
```

and below you can have `@SETTINGS["FW_BOOT_FULL_INIT"] = INIT_MAPPING[new_state]`